tiled/server/pydantic_array.py

```diff
@@ -27,6 +28,7 @@ class BuiltinDtype(BaseModel):
     endianness: Endianness
     kind: Kind
     itemsize: int
+    units: str = ""
```

Comment:
Would this be better done as `None` as the missing sentinel?  I worry about the in-band encoding of "no units because no one has said anything about what my units are, be warned they cloud be ANYTHING" as an empty string vs "I have units and the are an empty string because I am a dimensions value like the Rayleigh number" as an empty string.

Reply:
Thank you, I really like the idea! Changed the default to `None`.